repo/nirepo.yml

```diff
@@ -0,0 +1,1590 @@
+                  example: 5c62bf36781e2b1c7c57daf4
+      responses:
+        202:
+          description: OK
+          headers:
+            Location:
+              type: string
+              description: Contains the relative URI of the job that was created for tracking the status of the operation. Execute a GET against this URI to get information about the status of the operation.
+          schema:
+            $ref: '#/definitions/JobID'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/packages:
+    get:
+      x-ni-privilege: Read
+      x-ni-operation: listPackages
+      tags: [packages]
+      summary: Get all packages
+      description: Gets all of the packages in the service's package pool.
+      parameters:
+        - in: query
+          name: omitAttributes
+          type: boolean
+          description: If true, the "Attributes" dictionary for the package will be omitted. Defaults to false.
+        - in: query
+          name: omitFeedReferences
+          type: boolean
+          description: If true, the array of feed references will be omitted from each package. Defaults to false.
+        - in: query
+          name: returnUnreferencedOnly
+          type: boolean
+          description: A query parameter that determines if only packages without referencing feeds should be returned. Defaults to false.
+      responses:
+        200:
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Package'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/packages/{id}:
+    parameters:
+      - in: path
+        name: id
+        type: string
+        required: true
+    get:
+      x-ni-privilege: Read
+      x-ni-operation: getPackage
+      tags: [packages]
+      summary: Get package details.
+      description: Get package details.
+      parameters:
+        - in: query
+          name: omitAttributes
+          type: boolean
+          description: If true, the "Attributes" dictionary for the package will be omitted. Defaults to false.
+          required: false
+        - in: query
+          name: omitFeedReferences
+          type: boolean
+          description: If true, the array of feed references will be omitted from the package. Defaults to false.
+          required: false
+      responses:
+        200:
+          description: OK - The requested Package object. Not all of the fields on package objects will be available depending on the package. Not all fields will have content because not all packages will store this information. Do not rely on all of these fields being available in any application as some may be null, and empty string, or not exist of the object. However, key properties such as _id, name, owners, and FileName will never be null.
```

Comment:
I mean, it's fine if there is a rule for this. Otherwise it's weird to enter line breaks where there shouldn't be any, since this is a whole paragraph. I'm not aware of a way to do this in code without actually putting line breaks also in the documentation.

Reply:
There isn't a rule that I know of, but long unbroken lines diminish the maintainability of this document. They can't be viewed in GitHub without scrolling horizontally. Paul had asked me to break up similar lines in the alarm doc. I don't see a problem with putting line breaks in our swagger source.